_posts/en/newsletters/2018-08-14-newsletter.md

```diff
@@ -35,7 +35,22 @@ projects.
 
 ## Dashboard items
 
-<!-- TODO: @moneyball -->
+{% assign img1_label = "Transactions per block, 25-block moving average, July 14, 2018 - August 13, 2018" %}
+
+- Transaction [fees remain very low][fee metrics]: Anyone who can wait 10 or
+more blocks for confirmation can reasonably pay the default minimum fee rate.
+It’s a good time to [consolidate UTXOs][consolidate info].
+
+- [BTC hash rate][btc hash rate] briefly touched 60 EH/s on August 10, and has a 7 day average of 48 EH/s.
+
+- One way to measure activity in Bitcoin is to look at the the number of transaction in each 
```

Comment:
I don't know how I feel about this measurement given that transactions can be different sizes; I think counting inputs and outputs per block (separately) would be more informative.  Maybe the first sentence could be chopped off here to isolate this statistic from the idea that it has any implications, e.g. start the paragraph: "The number of transactions in each block has been vaguely periodic recently with peaks around ..."

Reply:
Block size (25 block moving average)
![block-size-25-ma](https://user-images.githubusercontent.com/12243734/44058982-e12bf92a-9f1d-11e8-84b9-64a891105172.png)

versus

Number of transactions (25 block moving average)
![transactions-spikes](https://user-images.githubusercontent.com/12243734/44058983-e137126a-9f1d-11e8-9402-521c49ed72e9.png)

Reply:
I made the suggested change to drop the initial portion of the first sentence.

Reply:
@marcinja Although block size in bytes is interesting from bandwidth and historical perspectives, block size in weight units or vbytes is what matters now, and it should be the case that (as long as there's a mempool backlog of > a few thousand txes), it'll always be near the max of 1 million vbytes / 4 million weight units.  For example, using this command on the last thousand blocks:

    for i in `seq 535664 536664` ; do echo -n "$i " ; bitcoin-cli getblock $( bitcoin-cli getblockhash $i ) | jq .weight ; done > block-weights.data

Produces this plot (no averaging):

![2018-08-13-21_10_08_154850265](https://user-images.githubusercontent.com/61096/44066292-81af7c5e-9f3d-11e8-887a-30030a3d3307.png)

Notice the clustering at the top end of the range.  The fairly even distribution of points below that top likely indicates periods where the mempool didn't have enough transactions for miners to fill out their blocks in an interblock period, so the fullness of the block depends mostly on how long it was since the previous block (random).  This is why some people (including me) think the default minimum relay fee is set too high---if it were lower, miners would create more full-sized blocks.